addon/components/frost-list-core.js

```diff
@@ -39,7 +36,6 @@ const FrostList = Component.extend(PropTypeMixin, {
   },
 
   alwaysUseDefaultHeight: false,
-  defaultHeight: 45,
```

Comment:
Is this no longer necessary (and why)?  Also, if it's still a useful mechanism, can it be set from the component interface?